src/Illuminate/Support/Str.php

```diff
@@ -356,6 +356,17 @@ public static function finish($value, $cap)
         return preg_replace('/(?:'.$quoted.')+$/u', '', $value).$cap;
     }
 
+    /**
+     * Convert a value to flat case.
+     *
+     * @param  string  $value
+     * @return string
+     */
+    public static function flat($value)
+    {
+        return Str::slug($value, '');
```

Comment:
you should probably write it as `static::slug`, that would keep it more in line with how `kebab()` works.